content/6_Advanced/XOR_Basis.mdx

```diff
@@ -8,6 +8,33 @@ frequency: 1
 
 export const problems = {
 	general: [
+		new Problem(
+			'AC',
+			'XOR Battle',
+			'https://atcoder.jp/contests/agc045/tasks/agc045_a',
+			'Medium',
```

Comment:
"Medium" difficulty doesn't exist. You should check how this renders in liveupdate.